pkg/services/mqtt/mqtt.go

```diff
@@ -0,0 +1,108 @@
+package mqtt
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/url"
+
+	"github.com/containrrr/shoutrrr/pkg/format"
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+
+	"github.com/containrrr/shoutrrr/pkg/services/standard"
+	"github.com/containrrr/shoutrrr/pkg/types"
+)
+
+const (
+	maxlength = 4096
+)
+
+// Service sends notifications to mqtt topic
+type Service struct {
+	standard.Standard
+	config *Config
+	pkr    format.PropKeyResolver
+}
+
+// Send notification to mqtt
+func (service *Service) Send(message string, params *types.Params) error {
+	if len(message) > maxlength {
+		return errors.New("message exceeds the max length")
+	}
+
+	config := *service.config
+	if err := service.pkr.UpdateConfigFromParams(&config, params); err != nil {
+		return err
+	}
+
+	return publishMessageToTopic(message, &config)
+}
+
+// Initialize loads ServiceConfig from configURL and sets logger for this Service
+func (service *Service) Initialize(configURL *url.URL, logger *log.Logger) error {
+	service.Logger.SetLogger(logger)
+	service.config = &Config{
+		DisableTLS:    true,
```

Comment:
Don't disable security features per default. It should need to be explicitly done if the user really needs to do so.